packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -194,6 +194,13 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
         commands.registerCommand({ id: 'workbench.action.files.newUntitledFile' }, {
             execute: () => open(this.openerService, createUntitledURI())
         });
+
+        if (isOSX && !environment.electron.is()) {
```

Comment:
I think you got the condition mixed up, it should be `!environment.electron.is() || isOSX`.

We want to contribute the command if we are in the browser version since our dialog supports the feature, and when we are running Electron we need to check if the system is OSX.

Reply:
You are right, thank you very much!